mods/bones/init.lua

```diff
@@ -5,8 +5,22 @@
 
 -- Load support for MT game translation.
 local S = minetest.get_translator("bones")
-
-bones = {}
+-- bones are supposed to hold up to 4*8+6+4*3*8+4+3*3 item slots:
+-- 4*8 for the main inventory
+-- 6 for the 3d_armor
+-- (at most) 4*3*8 for 4 backpack worth of items (unified inventory)
+-- 4 more for the actual backpacks
+-- 3*3 more for the crafting grid
+-- that adds up to 147, so 150 slots would be sufficient
+local cols=15
+local rows=10
+
+bones = {
+	private={
+		dead_player_callbacks={}
```

Comment:
Just use a local variable. This will also make the code considerably shorter and slightly faster.